# Upstream commit: https://github.com/gohugoio/hugo/commit/52edea0
# Trimmed down to patch only the stuff we need.

From 52edea0feccf98700300e98567c5a2ada7604c89 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bj=C3=B8rn=20Erik=20Pedersen?=
 <bjorn.erik.pedersen@gmail.com>
Date: Fri, 27 May 2022 10:06:45 +0200
Subject: [PATCH] github: Set HUGO_BUILD_TAGS: extended when running tests

Also fix TestDecodeConfig/Basic which started to fail in the extended build in 0.99.1.

Closes #9935
---
diff --git a/modules/config_test.go b/modules/config_test.go
index 55d055dcc7d..371aab05617 100644
--- a/modules/config_test.go
+++ b/modules/config_test.go
@@ -48,7 +48,7 @@ func TestDecodeConfig(t *testing.T) {

 [module.hugoVersion]
 min = "0.54.2"
-max = "0.99.0"
+max = "0.199.0"
 extended = true

 [[module.mounts]]
diff --git a/resources/resource_transformers/tocss/scss/integration_test.go b/resources/resource_transformers/tocss/scss/integration_test.go
index 72c0fd988e9..13b664cc723 100644
--- a/resources/resource_transformers/tocss/scss/integration_test.go
+++ b/resources/resource_transformers/tocss/scss/integration_test.go
@@ -236,7 +236,7 @@ T1: {{ $r.Content }}
 			}).BuildE()

 		b.Assert(err, qt.IsNotNil)
-		b.Assert(err.Error(), qt.Contains, filepath.FromSlash(`assets/scss/components/_foo.scss:2:1": expected ':' after $foocolor in assignment statement`))
+		b.Assert(err.Error(), qt.Contains, `assets/scss/components/_foo.scss:2:1": expected ':' after $foocolor in assignment statement`)
 		fe := b.AssertIsFileError(err)
 		b.Assert(fe.ErrorContext(), qt.IsNotNil)
 		b.Assert(fe.ErrorContext().Lines, qt.DeepEquals, []string{"/* comment line 1 */", "$foocolor #ccc;", "", "foo {"})
